src/jorphan/org/apache/jorphan/math/IStatCalculator.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.jorphan.math;
+
+import java.util.Map;
+
+public interface IStatCalculator<T extends Number & Comparable<? super T>> {
+    void clear();
+
+    /**
+     * Add to received bytes
+     * 
+     * @param newValue number of newly received bytes
+     */
+    void addBytes(long newValue);
+
+    /**
+     * Add to sent bytes
+     * 
+     * @param newValue number of newly sent bytes
+     */
+    void addSentBytes(long newValue);
```

Comment:
This looks like a violation of "single responsibility principle".
How `getStandardDeviation` is related to `addSentBytes`?

I think methods like `addSentBytes`, `addBytes`, `getTotalBytes`, `getTotalSentBytes` do not belong to a generic `IStatCalculator` interface.

Reply:
Agreed

Reply:
Is "getStandardDeviation" related to sentBytes? In my view the standardDeviation is for the response time of Http Request,  and "sent bytes" is for the amount of data used for transmission